### PR TITLE
docs/ppdse 2173 build foundation pages

### PR DESCRIPTION
**Pull request by khadirkilo92**
PPDSC-2173

**What**

1. Background - For documentation site to be up to date. 
2. What did you do: We created new folders on the theme page and added descriptions to our three new foundations so that it can be viewed on our live website. It is now in our routes as cards. 
3. What does the reviewers should expect: Addition to our routes with three new foundations on our theme page. Also created and uploaded the image files from design team. 

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [X ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on: 
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdse-2173-build-foundation-pages 
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdse-2173-build-foundation-pages/storybook

File: site/components/table/column-renderer.tsx [hunk 1]
```diff
@@ -214,6 +215,29 @@ export const renderCols = (
         cellContent = cellValue && (
           <BorderCard boxShadowToken={cellValue.toString()} />
         );
+        // console.log(<BorderCard boxShadowToken={cellValue.toString()} />);
```
Comment (mutebg) [inline site/components/table/column-renderer.tsx:218]: you might want to remove these comments

**Review by mutebg — COMMENTED** [review]
Well done @khadirkilo92 looks very good and you did 3 pages at the same time!

**Comment by Srinivasan-Ramasamy** [comment]
@khadirkilo92  Awesome work :+1:    

Thanks @baburay23 for guiding him.

**Review by Srinivasan-Ramasamy — APPROVED** [review]

**Review by Srinivasan-Ramasamy — APPROVED** [review]

**Pull request merged**